Makefile

```diff
@@ -97,15 +95,15 @@ test:
 
 .PHONY: build_docs
 build_docs:
-	podman run -v $(CURDIR)/docs:/docs:Z --rm $(DOCS_BUILD_CONTAINER) -b html5 -D /$(DOCS_BUILD_DIR) -o index.html $(DOCS_BUILD_TARGET)
+	podman run -v $(CURDIR)/docs:/docs:Z --rm $(DOCS_BUILD_CONTAINER) build_docs -b html5 -D $(DOCS_BUILD_DIR) -o index.html $(DOCS_BUILD_TARGET)
 
 .PHONY: build_docs_pdf
 build_docs_pdf:
-	podman run -v $(CURDIR)/docs:/docs:Z --rm $(DOCS_PDF_CONTAINER) -D /$(DOCS_BUILD_DIR) -o doc.pdf $(DOCS_BUILD_TARGET)
+	podman run -v $(CURDIR)/docs:/docs:Z --rm $(DOCS_BUILD_CONTAINER) build_docs_pdf -D $(DOCS_BUILD_DIR) -o doc.pdf $(DOCS_BUILD_TARGET)
 
 .PHONY: docs_serve
 docs_serve: build_docs
-	podman run -v $(CURDIR)/docs:/docs:Z --rm -p 8088:8088/tcp $(DOCS_SERVE_CONTAINER)
+	podman run -it -v $(CURDIR)/docs:/docs:Z --rm -p 8088:8088/tcp $(DOCS_BUILD_CONTAINER) docs_serve
```

Comment:
why `-it` now?

Reply:
It should've been there from before, we need the stdin open to send a `^C` to kill the server, `-t` is probably not needed here, unlike in https://github.com/code-ready/crc/pull/1499#discussion_r481845232 where we can get a more nicer colored output because of this.

Reply:
hmm ok, separate commit with just that change would have made sense, as here it's not clear if it's intentional or not, nor why this is done.